android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -610,6 +614,16 @@ private void close(final Result result) {
         result.success(null);
     }
 
+    private void setLogHandler(final Result result) {
+        CommonKt.setLogHandler(logData -> {
+            if (channel != null && this.activity != null) {
+                this.activity.runOnUiThread(() -> channel.invokeMethod(LOG_HANDLER_EVENT, logData));
```

Comment:
Not doing this causes an exception complaining that invokeMethod needs to be called on the UiThread

Reply:
I didnt' notice! but probably worth it to change threads there. I will add it